cli/commands/update.go

```diff
@@ -139,5 +135,7 @@ func HandleUpdate(app *kingpin.Application) {
 	update := app.Command("update", "Update the package version or configuration for this DC/OS service").Action(updateCmd.UpdateConfiguration)
 	update.Flag("options", "Path to a JSON file that contains customized package installation options").StringVar(&updateCmd.OptionsFile)
 	update.Flag("package-version", "The desired package version").StringVar(&updateCmd.PackageVersion)
-	update.Flag("status", "View status of this update").BoolVar(&updateCmd.Status)
+
+	status := update.Command("status", "View status of the current update").Alias("show").Action(updateCmd.PrintStatus)
```

Comment:
Oh I've seen that before. I think it's something to do with the arg handling library we're using, where it effectively runs _both_ commands. I'll see if there's a way to get around that...